sdk/src/main/scala/io/horizen/account/AccountSidechainNodeViewHolder.scala

```diff
@@ -245,4 +253,4 @@ object AccountNodeViewHolderRef {
 
 }
 
-case class MempoolReAddedTransactions[T <: SidechainTypes#SCAT](readdedTxs: Seq[T]) extends NodeViewHolderEvent
\ No newline at end of file
+case class NewExecTransactionsEvent(newExecTxs: Iterable[SidechainTypes#SCAT]) extends NodeViewHolderEvent
```

Comment:
let's put this class as a part of `object AccountSidechainNodeViewHolder`

Reply:
Done